scala/spark-wordcount/README.md

```diff
@@ -2,27 +2,41 @@
 
 ## Overview
 
-This sample demonstrates how to use  [Google Cloud Dataproc](https://cloud.google.com/dataproc), which
-provides a managed [Apache Spark](https://spark.apache.org/) environment with
-[Google Cloud Bigtable](https://cloud.google.com/bigtable/docs).
+This sample demonstrates how to use [Cloud Bigtable](https://cloud.google.com/bigtable) with [Apache Spark](https://spark.apache.org/) using [Dataproc](https://cloud.google.com/dataproc) (which
+provides a managed Apache Spark environment).
 
 This specific example provides the classic map reduce example of counting words
-in a text file.
+in a text file using [RDD API](https://spark.apache.org/docs/latest/rdd-programming-guide.html).
+
+**NOTE:** [Dataproc Bigtable Spark-HBase Connector Example](../bigtable-shc) is an example project that shows how to use [DataFrame API](https://spark.apache.org/docs/latest/sql-programming-guide.html) for structured data processing.
+
+There are two ways to deploy the sample Spark application:
+
+1. For production usage we recommend using [Dataproc](https://cloud.google.com/dataproc)
+1. For local testing use [Cloud Bigtable Emulator](https://cloud.google.com/bigtable/docs/emulator)
```

Comment:
This is an odd split: 1 talks about the execution env while 2 talks about the backend data store.

Maybe something like: 1. For development we recommend to use the local spark runner and the bigtable emulator 2. For production we recommend dataproc and a real Cloud Bigtable instance.

Reply:
fixed